snuba/migrations/migration.py

```diff
@@ -87,3 +91,12 @@ def forwards(self, context: Context) -> None:
             op.execute()
         logger.info(f"Finished: {migration_id}")
         update_status(Status.COMPLETED)
+
```

Comment:
Please remove the comment above 

```
TODO: This class should be extended to also run the forwards and backwards distributed
    operations where relevant.	    operations where relevant.
```

Reply:
Sorry, I didn't follow. Doesn't that TODO still apply? (it's about the distributed operations)

Reply:
I interpreted that comment as "add forward_dist and backwards_dist methods" or did that mean something else ?

Reply:
The methods were added but aren't used yet. I meant it to mean that we should also call execute on them in future